druid-shell/src/window.rs

```diff
@@ -141,7 +141,7 @@ impl WindowHandle {
     /// Sets the state of the window.
     ///
     /// [`state`]: enum.WindowState.html
-    pub fn set_window_state(&mut self, state: WindowState) {
+    pub fn set_window_state(&self, state: WindowState) {
```

Comment:
It seems odd to do this as a backend may not use interior mutability (especially if we had a completely virtual backend for tests etc).
Does it work on every platform?

Reply:
Hmm, having mutable here made the borrow checker complain

Reply:
I will revert this, maybe something to look into in the future when all backends are supported

Reply:
Anything set_* not taking &mut self or self seems suspect to me... (even if all current back ends do use interior mutability, it logically makes little sense)